lib/App/Netdisco/Worker/Plugin/Discover/PortProperties.pm

```diff
@@ -10,6 +10,7 @@ use Dancer::Plugin::DBIC 'schema';
 use Encode;
 use App::Netdisco::Util::FastResolver 'hostnames_resolve_async';
 use App::Netdisco::Util::Device qw/is_discoverable match_to_setting/;
+use App::Netdisco::Util::PortAccessEntity qw/update_pae_attributes/;
```

Comment:
don't know why this is here ... from an older version where it wasn't a separate plugin?

Reply:
exactly, gone now